make devel : missing server.key

On CentOS 7 a `make devel` in source directory will fail on rule `conf/ssl/server.crt`:
```
make: Circular conf/ssl/server.crt <- conf/ssl/server.crt dependency dropped.
openssl req -new -x509 -days 365 \
-out /usr/local/pf/conf/ssl/server.crt \
-key /usr/local/pf/conf/ssl/server.key \
-config /usr/local/pf/conf/openssl.cnf
Error opening Private Key /usr/local/pf/conf/ssl/server.key
140060599236496:error:02001002:system library:fopen:No such file or directory:bss_file.c:402:fopen('/usr/local/pf/conf/ssl/server.key','r')
140060599236496:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:404:
unable to load Private Key
make: *** [conf/ssl/server.crt] Error 1
```
I think we need to call `conf/ssl/server.key` rule before to avoid this error.

--- a/Makefile
+++ b/Makefile
@@ -165,7 +165,7 @@ systemd:
 pf-dal:
 	perl /usr/local/pf/addons/dev-helpers/bin/generator-data-access-layer.pl
 
-devel: configurations conf/ssl/server.crt conf/local_secret bin/pfcmd raddb/certs/server.crt sudo translation mysql-schema raddb/sites-enabled fingerbank chown_pf permissions bin/ntlm_auth_wrapper conf/unified_api_system_pass
+devel: configurations conf/ssl/server.key conf/ssl/server.crt conf/local_secret bin/pfcmd raddb/certs/server.crt sudo translation mysql-schema raddb/sites-enabled fingerbank chown_pf permissions bin/ntlm_auth_wrapper conf/unified_api_system_pass
 
 test:
 	cd t && ./smoke.t

